src/app/App.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import Dashboard from './pages/dashboard/Dashboard';
+
+function App(): JSX.Element {
+  return (
+    <>
```

Comment:
You don't need those fragments, at this point. Your Dashboard already returns fragments https://reactjs.org/docs/fragments.html